dff/script/core/actor.py

```diff
@@ -395,10 +396,10 @@ def validate_script(self, verbose: bool = True):
             response_func = normalize_response(node.response)
             try:
                 response_result = response_func(ctx, actor)
-                if isinstance(response_result, Callable):
+                if not isinstance(response_result, Message):
```

Comment:
Response can be callable in cases when callable response returns `Message`

Reply:
But here we check the type of `response_result` which is the return value of `response_func` (`Callable[[Context, Actor], Message]`), so `response_result` should be `Message`.

Reply:
Yes, you are right